doc/source/building/build_pxe_root_filesystem.rst

```diff
@@ -105,9 +105,9 @@ system. As diskless client, a QEMU virtual machine is used.
 
    .. code:: bash
 
-       DEFAULT KIWI-Boot
+       DEFAULT {kiwi}-Boot
```

Comment:
I guess the change doesn't really make a difference here. However I'd prefer to not use a variable here.